src/json_rpc/websocket_server.rs

```diff
@@ -106,7 +106,7 @@
 //! number of clients (the complexity being `O(n)`).
 //!
 //! The single-threadedness can even be beneficial when the WebSocket server isn't the primary
-//! *raison d'être* of a certain program, which is what this module has been designed for. In the
+//! *`raison d'être`* of a certain program, which is what this module has been designed for. In the
```

Comment:
```suggestion
//! purpose of a certain program, which is what this module has been designed for. In the
```

https://en.wiktionary.org/wiki/raison_d%27%C3%AAtre

I thought I could be fancy by using a bit of French